static/templates/settings/notification-settings.handlebars

```diff
@@ -72,16 +82,28 @@
               "is_checked" page_params.enable_offline_email_notifications
               "label" settings_label.enable_offline_email_notifications}}
 
-            {{partial "settings_checkbox"
-              "setting_name" "enable_offline_push_notifications"
-              "is_checked" page_params.enable_offline_push_notifications
-              "label" settings_label.enable_offline_push_notifications}}
-
-            <div class="input-group disableable {{#unless page_params.enable_offline_push_notifications}}control-label-disabled{{/unless}}">
+            {{!-- Mobile notifications when offline --}}
+            <div class="input-group {{#unless page_params.realm_push_notifications_enabled}}control-label-disabled{{/unless}}" id="push-warning">
```

Comment:
`id` used here is redundant and is repeated here as it's already being used above.